config.cjs

```diff
@@ -0,0 +1,232 @@
+
+env.addFilter('log', function(x) { console.log(x); });
+
+StyleDictionary.registerFileHeader({
+  name: 'redhat/legal',
+  fileHeader(defaultMessage) {
+    return [
+      ...defaultMessage,
+      '',
+      '@license',
+      ...fs.readFileSync(path.join(__dirname, 'LICENSE'), 'utf8').split('\n'),
+    ];
+  },
+});
+
+StyleDictionary.registerTransform({
+  type: 'value',
+  name: 'shadow/css',
+  matcher: token => token.path.includes('box-shadow'),
+  transformer: ({ value: { offsetX, offsetY, blur, spread, color }}) =>
+    `${offsetX} ${offsetY} ${blur} ${spread} ${color}`,
+})
+
+StyleDictionary.registerTransformGroup({
+  name: 'css',
+  transforms: [
+    'shadow/css',
+    'attribute/cti',
+    'name/cti/kebab',
+    'time/seconds',
+    'content/icon',
+    'size/rem',
+    'color/css',
+  ]
+})
+
+StyleDictionary.registerTransformGroup({
+  name: 'scss',
+  transforms: [
+    'shadow/css',
+    'attribute/cti',
+    'name/cti/kebab',
+    'time/seconds',
+    'content/icon',
+    'size/rem',
+    'color/css',
+  ]
+})
+
+StyleDictionary.registerFormat({
+  name: 'css/lit',
+  formatter: ({ file, dictionary, options }) => `${fileHeader({file})}
+import { css } from 'lit';
+export const resetStyles = css\`
+:host {
+${formattedVariables({ format: 'css', dictionary, outputReferences: options.outputReferences })}
+}\`;
+export default resetStyles;`,
+});
+
+StyleDictionary.registerFormat({
+  name: 'snippets/vscode',
+  formatter: ({ dictionary }) =>
+    JSON.stringify(Object.fromEntries(
+      dictionary.allProperties.map(prop => [
+        prop.title ?? prop.name,
+        {
+          scope: 'css,scss',
+          // prefix: prop.path.reduce((prefixes, path) => [...prefixes, `${prefixes.at(-1)}-${path}`], ['--rh']),
+          prefix:[  `--${prop.name}`],
+          body: [`var(--${prop.name}, ${prop.value})`],
+          description: prop.comment,
+        }
+      ])
+    ), null, 2),
+});
+
+StyleDictionary.registerFormat({
+  name: 'html/docs',
+  formatter({ dictionary, platform, options }) {
+    return env.render('index.html', {
+      dictionary,
+      platform,
+      colors: dictionary.allTokens.filter(x => x.attributes?.category === 'color').reduce((acc, color) => ({
+        ...acc,
+        [color.attributes.type]: [
+          ...acc[color.attributes.type] ?? [],
+          color,
+          ...Object.entries(color).flatMap(([key, value]) =>
+            typeof value !== 'object' || key.match(/original|attributes|path/)? [] : ({
+              ...value,
+              name: value.name ?? color.path.reduce((a, b)=> `${a}-${b}`, 'rh') + '-' + key,
+            }))
+        ],
+      }), {}),
+    })
+  }
+});
+
+module.exports = {
+  source: [
+    'tokens/**/*.{yaml,yml}',
+  ],
+  parsers: [{
+    pattern: /\.ya?ml$/,
+    parse({ contents }) {
+      return YAML.parse(contents);
+    }
+  }],
+  platforms: {
+    css: {
+      transformGroup: 'css',
+      buildPath: 'build/css/',
```

Comment:
Should we build into the repo root and .gitignore the artifacts? that would simplify loading css from cdn urls

Reply:
I think so.

Reply:
done